packages/expo-asset/src/__tests__/Asset-test.ts

```diff
@@ -27,7 +27,7 @@ jest.mock('../ImageAssets', () => {
   const ImageAssets = jest.requireActual('../ImageAssets');
   return {
     ...ImageAssets,
-    getImageInfoAsync: jest.fn(),
+    getImageInfoAsync: jest.fn(() => Promise.resolve({ width: 1, height: 1 })),
```

Comment:
Doesn't matter much, just pointing out that I like to mock async functions with async functions themselves:

```suggestion
    getImageInfoAsync: jest.fn(async () => ({ width: 1, height: 1 })),
```